src/components/templates/dao/DAO.tsx

```diff
@@ -0,0 +1,22 @@
+import { Heading, VStack } from '@chakra-ui/react';
+import { Window, Launcher } from '@relaycc/receiver';
+import { Wallet } from 'ethers';
+const w = Wallet.createRandom();
+
+
+const DAO = (props:any) => {
```

Comment:
Instead of `(props: any)`, use `(props: Props)` after you create the interface

Also, you shouldn't be using `any` - read https://thoughtbot.com/blog/typescript-stop-using-any-there-s-a-type-for-that